### PR TITLE
adding trochlear ridge of humerus

### PR DESCRIPTION
**Pull request by megbalk**
Issue #1878

### CODE REVIEW

**Review by nicolevasilevsky — APPROVED** [review]

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224701,6 +224701,15 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
 intersection_of: connects UBERON:7500048 ! proximal-most point of ventral tubercle of humerus
 created_by: https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500061
+name: trochlear ridge of humerus
+def: "The ridge of the trochlea on a humerus lies in the middle of the distal end of a humerus on the lateral side of the trochlea and the medial side of the capitulum (capitellum)." [ORCID: 0000-0003-2699-3066]
+intersection_of: UBERON:0004765 ! skeletal element
+intersection_of: part_of UBERON:0000144 ! trochlea of humerus
```
Comment (dosumis) [inline src/ontology/uberon-edit.obo:224709]: See https://github.com/obophenotype/uberon/pull/1897/files#r676113805 for suggested change to classification.

**Review by dosumis — CHANGES_REQUESTED** [review]
Requested classification change

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (dosumis) [reply src/ontology/uberon-edit.obo:224709]: I think you said you'd done this in a comment on #1897 (?)Looks like this change still needs to be pushed - or is it on the other PR?

Comment (megbalk) [reply src/ontology/uberon-edit.obo:224709]: done.

**Review by dosumis — APPROVED** [review]

**Pull request merged**